.golangci.yaml

```diff
@@ -1,7 +1,6 @@
 linters:
   disable-all: true
   enable:
-    - depguard
```

Comment:
`depguard` was reporting erroneous results and, without configuration, doesn't provide much value. I'm choosing to remove it instead of figuring out what caused it to break.